### PR TITLE
Dynamic docs version

### PR DESCRIPTION
**Pull request by ben-bay**
Our readthedocs is still showing the version as `1.0.0`. This should fix that.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -16,17 +16,18 @@
 import sys
 sys.path.insert(0, os.path.abspath('../../..'))
 
+MERLIN_VERSION = __import__("merlin").VERSION
 
 # -- Project information -----------------------------------------------------
 
 project = u'Merlin'
-copyright = u'2018, MLSI'
+copyright = u'2019, MLSI'
```
Comment (lucpeterson) [inline docs/source/conf.py]: Can change to
```
from datetime import date
_year = date.today().year
copyright = f'{_year}, MLSI'
```

Comment (lucpeterson) [reply docs/source/conf.py]: this way we won't have to change it again in a few weeks

File: docs/source/conf.py [hunk 2]
```diff
@@ -16,17 +16,18 @@
...
 author = u'MLSI'
 
 # The short X.Y version
-version = u'1.0.0'
+version = MERLIN_VERSION
 # The full version, including alpha/beta/rc tags
-release = u'1.0.0'
+release = MERLIN_VERSION
```
Comment (lucpeterson) [inline docs/source/conf.py]: do these have to be strings?

Comment (ben-bay) [reply docs/source/conf.py]: You want it to be a unicode string?

**Pull request merged**